accountsdb-plugin-postgres/scripts/create_schema.sql

```diff
@@ -113,27 +113,28 @@ CREATE TYPE "TransactionMessage" AS (
     instructions "CompiledInstruction"[]
 );
 
-CREATE TYPE "AddressMapIndexes" AS (
-    writable SMALLINT[],
-    readonly SMALLINT[]
+CREATE TYPE "TransactionMessageAddressTableLookup" AS (
```

Comment:
Is the TransactionMessageAddressTable a global construct or stored in the transaction itself? If it is the former, we will have to persist that global table as well otherwise the indexes cannot be used.

Reply:
> Is the TransactionMessageAddressTable a global construct

It's stored on-chain so the full address table won't be included in the transaction object, but those addresses are resolved into "LoadedAddresses"